src/features/smartSearch/components/filters/Journey/DisplayJourney.tsx

```diff
@@ -27,19 +26,13 @@ const localMessageIds = messageIds.filters.journey;
 const DisplayJourney: FC<DisplayJourneyProps> = ({ filter }): JSX.Element => {
   const { orgId } = useNumericRouteParams();
   const op = filter.op || OPERATION.ADD;
-  const {
-    operator,
-    journey: journeyId,
-    after,
-    before,
-    tags: tagsObj,
-  } = filter.config;
+  const { journey: journeyId, tags: tagsObj } = filter.config;
   const journeys = useJourneys(orgId).data || [];
-  const journeyTitle = journeys?.find((item) => item.id === journeyId)?.title;
-  const timeFrame = getTimeFrameWithConfig({
-    after: after,
-    before: before,
-  });
+  const journeyTitle = journeys?.find(
+    (item) => item.id === journeyId
+  )?.plural_label;
```

Comment:
is it not risky to not give this a "backup" of some sort, like an empty string, since find cannot guarantee a journey will be found? Or is it safe enough that we can know that there will always be a journey here?

Reply:
It is a little bit risky, but not hugely. I didn't change that aspect from the original code, but I guess we should probably fix this here and in other parts of Smart Search.

The reason it's only a little bit risky, is that the `journeyTitle` will (because of the `?`) be typed as `string | undefined`. Because it's typed as possible undefined, I trust other parts of the code are careful about this (or TypeScript would complain), and my conclusion is that worst case scenario, the UI might present the string "undefined" somewhere. Not good, but at least it's not a crash.